.eslintrc.json

```diff
@@ -13,22 +13,47 @@
     "consistent-return": 0,
     "no-param-reassign": 0,
     "comma-dangle": 0,
-    "curly": ["error", "multi-line"],
-    "import/no-unresolved": [2, { "commonjs": true }],
-    "no-shadow": ["error", { "allow": ["req", "res", "err"] }],
-    "valid-jsdoc": ["error", {
-      "requireReturn": true,
-      "requireReturnType": true,
-      "requireParamDescription": false,
-      "requireReturnDescription": true
-    }],
-    "require-jsdoc": ["error", {
-      "require": {
-        "FunctionDeclaration": true,
-        "MethodDefinition": true,
-        "ClassDeclaration": true
+    "curly": [
```

Comment:
Is there any difference between your current changes and what is in the remote develop branch?

Reply:
Yeah, I added this:
"parserOptions": {
    "sourceType": "module"
  }
But other changes you see is due to formatting on save